### PR TITLE
Add finish variant to WebGLCommand

### PR DESCRIPTION
**Pull request by cbrewster**
This is for servo/servo#10212.

### CODE REVIEW

File: src/webgl.rs [hunk 1]
```diff
@@ -326,6 +328,8 @@ impl WebGLCommand {
                 sender.send(ctx.borrow_draw_buffer().unwrap().size().width).unwrap(),
             WebGLCommand::DrawingBufferHeight(sender) =>
                 sender.send(ctx.borrow_draw_buffer().unwrap().size().height).unwrap(),
+            WebGLCommand::Finish(sender) =>
+                { gl::finish(); sender.send(()).unwrap(); },
```
Comment (KiChjang) [inline src/webgl.rs:332]: r-

I'm not very fond of having multiple statements per line.

Comment (emilio) [inline src/webgl.rs:332]: I agree, but on the other hand, movin this to one new function seemed a bit overkill to me (and doing `sender.send(gl::finish()).unwrap()`, while it's an option, doesn't look right to me).

Anyways, can be changed:

@ConnorGBrewster could you ammend this in the upcomming `DetachShader` PR, moving it into a new function, if it's not too much problem?

Comment (cbrewster) [inline src/webgl.rs:332]: I will move that to a function in the `DetachShader` PR

Comment (KiChjang) [inline src/webgl.rs:332]: I meant that instead of doing a one-liner, we should split it up into multiple lines.

Comment (cbrewster) [inline src/webgl.rs:332]: I think moving it to a separate function makes it look more consistent with the rest of the match branches, but I can move it back and put the statements on 2 lines.

**Pull request merged**